spec/services/api/create_params_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::CreateParams do
+  describe "#permissions" do
+    it "allows scalar model attributes" do
+      pl_permissions = Api::CreateParams.new(Api::Model.new('plant_line'), {}).permissions
+      # FIXME plant_variety_name - shouldn't it be removed?
```

Comment:
Well, they are "documented" - in past GitHub issues. I leave it up to Piotr if and how he'd like to maintain a separate "curation pending" list.